38.JS_Async/assets/js/main.js

```diff
@@ -0,0 +1,201 @@
+const API_KEY = "7a4aabbe";
+const API_URL = `http://www.omdbapi.com/?apikey=${API_KEY}`;
+
+const fetchData = async (query, type, page = 1) => {
+  return await fetch(`${API_URL}&s=${query}&type=${type}&page=${page}`).then(
+    (response) => response.json()
+  );
+};
+
+const fetchDetails = async (id) => {
+  return await fetch(`${API_URL}&i=${id}`).then((response) => response.json());
+};
+
+const getSearchForm = () => {
```

Comment:
nice solution for practise but will be beater just use const and select elements into the top of file

Reply:
same for all functions